server/src/main/resources/org/opensearch/bootstrap/security.policy

```diff
@@ -182,4 +182,6 @@ grant {
   permission java.io.FilePermission "/sys/fs/cgroup/memory", "read";
   permission java.io.FilePermission "/sys/fs/cgroup/memory/-", "read";
 
+  // ZSTD and lz4-java permissions
+  permission java.lang.RuntimePermission "*";
```

Comment:
This is certainly a security loophole and defeats the usage of `SecurityManager`, we need to specify fine-grained `RuntimePermission` required by compressors.

Reply:
Modified it to use `loadLibrary.*` target.